shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/hint/SQLHintUtils.java

```diff
@@ -48,9 +53,12 @@ public static Properties getSQLHintProps(final String comment) {
         }
         startIndex = startIndex + SQL_HINT_TOKEN.length();
         int endIndex = comment.endsWith(SQL_COMMENT_SUFFIX) ? comment.indexOf(SQL_COMMENT_SUFFIX) : comment.length();
-        String[] hintValue = comment.substring(startIndex, endIndex).trim().split(SQL_HINT_SPLIT);
-        if (2 == hintValue.length && hintValue[0].trim().length() > 0 && hintValue[1].trim().length() > 0) {
-            result.put(hintValue[0].trim(), hintValue[1].trim());
+        Collection<String> sqlHints = Splitter.on(SQL_HINT_SPLIT).trimResults().splitToList(comment.substring(startIndex, endIndex).trim());
+        for (String each : sqlHints) {
+            List<String> hintValues = Splitter.on(SQL_HINT_VALUE_SPLIT).trimResults().splitToList(each);
+            if (2 == hintValues.size()) {
```

Comment:
why not use a meaningful constant here instead of magic number `2`

Reply:
> number

@taojintianxia Thank you, I will optimize it.